lib/pages/chat/chat.dart

```diff
@@ -1050,14 +1054,21 @@ class ChatController extends State<Chat>
     if (!mounted) {
       return;
     }
-    await scrollToIndex(eventIndex);
+    await scrollToIndex(eventIndex, highlight: highlight);
     _updateScrollController();
   }
 
-  Future scrollToIndex(int index) => scrollController.scrollToIndex(
+  Future scrollToIndex(int index, {bool highlight = false}) async {
+    await scrollController.scrollToIndex(
```

Comment:
i think `jumpToIndex` can be better for performance

Reply:
remove the long scroll animation

Reply:
or you can make it faster

Reply:
@drminh2807 may be you forgot this comment

Reply:
attempted before but it looks like UI freezed

Reply:
added loading, please see demo